src/main/java/zipkin/sparkstreaming/adjuster/BinaryAnnotationTrimAdjuster.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.sparkstreaming.adjuster;
```

Comment:
better to not use core package names. maybe `sparkstreaming.tagtruncator` which is short and to the point.

    On that note, a lot of users don't use the word binary annotation and we are actually changing the model to the more accessible term "tag"

    To make the example easier you could consider just using the word tag. That's what's used in opentracing, brave4 and the upcoming simplified model.

    In the description you can say that tag is a binary annotation of type string (which happens to be the only usable binary annotation).